src/python/pants/notes/master.rst

```diff
@@ -4,8 +4,33 @@ Master Pre-Releases
 This document describes ``dev`` releases which occur weekly from master, and which do
 not undergo the vetting associated with ``stable`` releases.
 
-1.8.0dev4 (06/15/2018)
```

Comment:
Note that there was a punctuation problem here, hence the edit.

Reply:
Thanks!